src/components/selector/selector.tsx

```diff
@@ -108,7 +109,7 @@ export const Selector = <V extends SelectorValue>(p: SelectorProps<V>) => {
     <div className={classPrefix}>
       {!props.columns && <Space wrap>{items}</Space>}
       {props.columns && (
-        <Grid columns={props.columns} gap={convertPx(8)}>
+        <Grid columns={props.columns} gap={props.gap || convertPx(8)}>
```

Comment:
```suggestion
        <Grid columns={props.columns} gap={props.gap}>
```

`convertPx(8)` 放到 `defaultProps` 吧

Reply:
ok~